Channels/Insider/release_notes.md

```diff
@@ -1,9 +1,26 @@
-## April 2019 Insider Release ([v1.1.0822.06](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0822.06))
+## April 2019 Insider Release ([v1.1.0828.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0828.01))
+
+***Note: This is a preliminary version that needs more detailed copy***
 
 Welcome to the April 2019 release of Accessibility Insights for Windows. Key highlights include:
 
-- [**GitHub issue filing**](#github-issue-filing): File GitHub issues directly from Accessibility Insights for Windows.
+  - [GitHub Issue Filing](#github-issue-filing)
+  - [Multiple Release Channels](#multiple-release-channels)
+  - [Rule Updates](#rule-updates)
+
+### GitHub Issue Filing
+
+GitHub users can now file issues directly from inside Accessibility Insights for Windows. Bug filing from Azure DevOps is still supported, and you can freely switch between the two.
+
+### Multiple Release Channels
+
+You can now choose how frequently their client updates:
+- The **Production** channel updates every 2-4 weeks
+- The **Insider** channel updates every 1-2 weeks
+- The **Canary** channel updates multiple times per week
+
+You'll be in the **Production** channel by default, but you can change your channel at any time via the **Settings** page
 
-### GitHub issue filing
+### Rule Updates
 
-GitHub users can now file issues directly from inside Accessibility Insights for Windows. (Need Copy here)
+We have updated several of our accessibility rules based on community feedback.
```

Comment:
do we have a list of rules?

Reply:
At this point, only a couple of rules have been modified. Rob and Karan are working on more, but they aren't included in the current release

Reply:
could you make sure that at least the two known user reported false positive fixes are included when a later build is promoted to Insider?